jobs/admin.py

```diff
@@ -22,7 +22,7 @@ def make_published(modeladmin, request, queryset):
 
 
 def send_status_update_job_offer(modeladmin, request, queryset):
-    option = 'job offer'
+    option = 'job'
```

Comment:
Not for this PR - but I feel the send_status_update methods could be refactored.  Most of the code for the job and the meetup version is identical and could be a single method that gets called from the separate job/meetup send_status_update method.